### PR TITLE
Cleanup or man pages

### PR DESCRIPTION
**Pull request by rhatdan**
We need to build from the md file on every use.  .1 file needs to be removed.

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -6,5 +6,11 @@ oci_systemd_hook_LDADD = $(YAJL_LIBS)
...
 
-dist_man_MANS = doc/oci_systemd_hook.1
+dist_man_MANS = oci_systemd_hook.1
 EXTRA_DIST = README.md LICENSE
+
+oci_systemd_hook.1: doc/oci_systemd_hook.1.md
+	go-md2man -in doc/oci_systemd_hook.1.md -out oci_systemd_hook.1
+
+clean-local:
+	-rm -f oci_systemd_hook.1
```
Comment (mrunalp) [inline Makefile.am:16]: typo: -rm -> rm

Comment (rhatdan) [inline Makefile.am:16]: I believe this is correct,  This tells make not to show the rm command.

Comment (mrunalp) [inline Makefile.am:16]: Ahh, okay. Today I learned :)

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**